src/pages/signup/index.js

```diff
@@ -66,7 +66,7 @@ const NewSignup = () => {
         setSubmitState(submitStatus)
     }
     return (
-        <Layout type="static" margin_top={0}>
+        <Layout type="static" margin_top={'0'}>
```

Comment:
Should `0` be a number instead of a string?

Reply:
there were usage where
```
margin_top && margin_top
```

if we use 0, it will return false, where '0' will be the correct props

Reply:
May need to adjust the PropType in the next PR as well.

Reply:
should it be changed to string or accept both string/number?

Reply:
You can set it to accept both 👍

Reply:
noted.